base/challenge-skeleton/healthcheck/Dockerfile

```diff
@@ -17,9 +17,6 @@ FROM kctf-healthcheck
 # This is out of caution in case that we leak the 
 COPY .gen/exploit.cpio.enc /
 
-# Needed by the webserver healthz.py
-RUN apt-get -y install python2.7
```

Comment:
this now is installed on `base/healthcheck-docker/Dockerfile` since it's necessary for pwntools anyway

Reply:
Hmm, pwntools is using the python from the venv and not the global one, that's why this should be installed here IMO.